inc/osvr/ClientKit/ContextC.h

```diff
@@ -49,7 +49,15 @@ OSVR_EXTERN_C_BEGIN
 
 /** @addtogroup ClientKit
     @{
-*/
+    */
+
+/** @brief Ensures the OSVR server process/thread is running. Call once per process.
+ */
+ OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrServerAutoStart();
+
+/** @brief Ensures any OSVR server-related resources in the client are released. Call once per process.
+ */
+OSVR_CLIENTKIT_EXPORT OSVR_ReturnCode osvrServerRelease();
```

Comment:
Multiple client contexts in existing applications mean this work can't be done in client context closing without internal bookkeeping (i.e. reference counting or similar).